docs/UserGuide.md

```diff
@@ -686,7 +709,7 @@ works. However, we would strongly recommend you not to as you may cause data cor
 | ---                                          | ---                                                      |
 | Help                                         | `help`                                                   |
 | Estimate time needed to translate the script | `estimate <SCRIPT_FILE_NAME> [-wph WORDS_PER_HOUR]`      |
-| Browse                                       | `browse [-s SORT_OPTION] [-p PAGE_NO.] [-o SORT_ORDER]`  |
+| Browse                                       | `browse [-s <SORT_OPTION>] [-p <PAGE_NO.>] [-o <SORT_ORDER>]`  |
```

Comment:
I think you don't need a <> inside the optional argument bracket, similarly for line 726.

Reply:
Thank you for pointing this out, I have rectified the issue, in the latest commit.